mod-data-import-converter-storage-server/src/main/java/org/folio/dao/association/AbstractProfileAssociationDao.java

```diff
@@ -82,15 +98,65 @@
   }
 
   @Override
-  public Future<D> getDetailProfilesByMasterId(String masterId) {
-    // STUB implementation, use detailProfileDao to obtain "detail" profiles
-    return Future.succeededFuture();
+  public Future<List<ChildSnapshotWrapper>> getDetailProfilesByMasterId(String tenantId, String masterId) {
+    return select(tenantId, format(RETRIEVES_DETAILS_SQL, masterId)).map(this::mapToDetails);
+  }
+
+  private List<ChildSnapshotWrapper> mapToDetails(ResultSet resultSet) {
```

Comment:
private methods also should have javadoc

Reply:
Done